QuizMakerAngular/src/app/app.component.css

```diff
@@ -0,0 +1,51 @@
+nav a {
+    padding: 5px 10px;
```

Comment:
This can be a future thing, but avoid using the 'px' sizing, rather use a relative sizing (e.g. rem), so that the page will be responsive no matter what your display size it

Reply:
noted